realm/src/androidTest/java/io/realm/RealmInMemoryTest.java

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
Not too much copy/paste here. It's only in-mem related cases here since the diff here is writing to the disk or not. But still case like write-copy-to is pretty the same with the on-disk-realm. Maybe it is still good idea to generalize all the cases we have in RealmTest to apply to different kinds of durability?